### ISSUE TITLE
Default branch will be renamed main

### ISSUE THREAD

**Issue by BillWagner** [dotnet/samples#4400]
On March 9th, 2021, we're going to rename the default branch in this repository to main. For more details, see [dotnet's earlier announcement](https://github.com/dotnet/announcements/issues/172).

**Comment by BillWagner** [comment]
The branch renaming is complete.

If you have a fork and/or local clone, here are the steps you need to take:

1. Navigate to your fork on GitHub, go to the Settings tab, then the Branches tab, and then under Default branch, click the pencil icon to rename the default branch to main.
2. If you have a local clone, run the following commands in Git Bash:

   ```
   git branch -m master main
   git fetch origin
   git branch -u origin/main main
   ```

**Comment by BillWagner** [comment]
Closed in #5512
